run-checks

```diff
@@ -55,6 +55,12 @@ case "${1:-all}" in
     --spread)
         SPREAD=1
         ;;
+    --spread-ubuntu)
+        SPREAD=2
+        ;;
+    --spread-no-ubuntu)
+        SPREAD=3
```

Comment:
And SPREAD=no-ubuntu here? This way the checks below will convey a bit more context.